x-pack/plugins/apm/public/components/app/service_overview/service_overview_charts/use_filters_for_mobile_charts.ts

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { useMemo } from 'react';
+import { type QueryDslQueryContainer } from '@elastic/elasticsearch/lib/api/types';
+import { isNil, isEmpty } from 'lodash';
+import { ProcessorEvent } from '@kbn/observability-plugin/common';
+import { environmentQuery } from '../../../../../common/utils/environment_query';
+import { useApmParams } from '../../../../hooks/use_apm_params';
+import {
+  SERVICE_NAME,
+  TRANSACTION_TYPE,
+  PROCESSOR_EVENT,
+  HOST_OS_VERSION,
+  DEVICE_MODEL_IDENTIFIER,
+  NETWORK_CONNECTION_TYPE,
+  SERVICE_VERSION,
+} from '../../../../../common/elasticsearch_fieldnames';
+
+function termQuery<T extends string>(
```

Comment:
Can't you use the one that already exists in the [Observability plugin](https://github.com/elastic/kibana/blob/main/x-pack/plugins/observability/server/utils/queries.ts#L20:L20)

Reply:
It's in the `/server`, the plan is to move it to a common folder on a follow up PR